lib/Config.js

```diff
@@ -243,6 +243,23 @@ class Config {
             }
         }
 
+        this.metadataDaemon = {};
```

Comment:
It's better to add defaults here since host and port are required for the file backend to start.
```js
this.metadataDaemon = { host: 'localhost', port: 9990 }
```